pkg/build/controller/policy/policy.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildclient "github.com/openshift/origin/pkg/build/client"
+	buildlister "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
```

Comment:
This is not moving to caches.  This is replacing the code that was using the dead hand written client.